tests/pages/externalpages/pocket/base.py

```diff
@@ -0,0 +1,101 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from pypom import Page, Region
+from selenium.webdriver.common.by import By
+
+
+class ScrollElementIntoView:
+    def scroll_element_into_view(self, strategy, locator):
+        # scroll elements so they are not beneath the navigation
+        offset = {"x": 0, "y": -100}
+        return super(ScrollElementIntoView, self).scroll_element_into_view(strategy, locator, **offset)
+
+
+class BaseRegion(ScrollElementIntoView, Region):
+    pass
+
+
+class BasePage(ScrollElementIntoView, Page):
+    def __init__(self, selenium, base_url, locale="en-US", **url_kwargs):
+        super(BasePage, self).__init__(selenium, base_url, locale=locale, **url_kwargs)
+
+    def wait_for_page_to_load(self):
+        browser_name = self.selenium.capabilities.get("browserName").lower()
+        browser_version = self.selenium.capabilities.get("browserVersion") or self.selenium.capabilities.get("version")
+        is_ie9 = browser_name == "internet explorer" and browser_version == "9"
+
+        self.wait.until(lambda s: self.seed_url in s.current_url)
+
+        # legacy IE browsers are not served JS, so they do not get the 'loaded' conditional class name.
+        if not is_ie9:
+            el = self.find_element(By.TAG_NAME, "html")
+            self.wait.until(lambda s: "loaded" in el.get_attribute("class"))
+
+        return self
+
+    @property
+    def URL_TEMPLATE(self):
+        if "{params}" in self._URL_TEMPLATE:
+            return f"{self._URL_TEMPLATE}&automation=true"
+        else:
+            return f"{self._URL_TEMPLATE}?automation=true"
+
+    @property
+    def navigation(self):
+        return self.Navigation(self)
+
+    class Navigation(BaseRegion):
+
+        _mobile_menu_open_btn_locator = (By.CLASS_NAME, "global-nav-mobile-menu-btn")
+        _mobile_menu_close_btn_locator = (By.CLASS_NAME, "mobile-nav-close-btn")
+        _mobile_menu_locator = (By.CLASS_NAME, "mobile-nav-list")
+        _home_mobile_menu_locator = (By.CSS_SELECTOR, '.mobile-nav-list-link[href="https://getpocket.com/home?src=navbar"]')
+        _my_list_mobile_menu_locator = (By.CSS_SELECTOR, '.mobile-nav-list-link[href="https://getpocket.com/my-list?src=navbar"]')
+        # _discover_mobile_menu_locator = (By.CSS_SELECTOR, '.mobile-nav-list-link[href="https://getpocket.com/explore?src=navbar"]')
```

Comment:
not sure if it's worth testing all the mobile links?

Reply:
Agreed - only asserting that some expected content is displayed / visible inside the nav when it opens should be enough. Also agree that we don't need to go around clicking the links here (since they open in prod and not locally)